app/main.py

```diff
@@ -1,25 +1,82 @@
-from abc import ABC
+from abc import ABC, abstractmethod
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __get__(self, instance: int, owner: object) -> None:
+        return getattr(instance, self._name)
+
+    def __set__(self, instance: object, value: int) -> None:
+        if not isinstance(value, int):
+            raise ValueError("Value must be an integer")
+        if not self.min_amount <= value <= self.max_amount:
+            raise ValueError(f"Value must be between {self.min_amount}"
+                             f" and {self.max_amount}")
+        setattr(instance, self._name, value)
+
+    def __set_name__(self, owner: object, name: str) -> None:
+        self._name = name
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self, age: "IntegerRange",
```

Comment:
Indicate here that you expect to obtain integer instead of an instance of `IntegerRange` class